tools/bumbershoot/custom_pro_db/tool-data/customProDB.loc.sample

```diff
@@ -1,18 +0,0 @@
-#This file lists the locations and dbkeys of all the fasta files
-#under the "genome" directory (a directory that contains a directory
```

Comment:
This is needed afaik just to make this toolshed repo aware of the data_table. You can not upload this tool to the toolshed without these files/structure.